test/testfixture/recipe_funcs.go

```diff
@@ -573,6 +573,37 @@ func Trackers(n int, fns ...CustomizeTrackerFunc) RecipeFunction {
 	}
 }
 
+// CustomizeTrackerQueriesFunc is directly compatible with CustomizeEntityFunc
+// but it can only be used for the Trackers() recipe-function.
+type CustomizeTrackerQueriesFunc CustomizeEntityFunc
+
+// TrackerQueries tells the test fixture to create at least n tracker objects. See
+// also the Identities() function for more general information on n and fns.
+//
+// When called in NewFixture() this function will call also call
+//     Spaces(1)
+// but with NewFixtureIsolated(), no other objects will be created.
+func TrackerQueries(n int, fns ...CustomizeTrackerQueriesFunc) RecipeFunction {
+	return func(fxt *TestFixture) error {
+		fxt.checkFuncs = append(fxt.checkFuncs, func() error {
+			l := len(fxt.TrackerQueries)
+			if l < n {
+				return errs.Errorf(checkStr, n, kindTrackerQueries, l)
+			}
+			return nil
+		})
+		// Convert fns to []CustomizeEntityFunc
+		customFuncs := make([]CustomizeEntityFunc, len(fns))
+		for idx := range fns {
+			customFuncs[idx] = CustomizeEntityFunc(fns[idx])
+		}
+		if err := fxt.setupInfo(n, kindTrackerQueries, customFuncs...); err != nil {
+			return err
+		}
+		return fxt.deps()
```

Comment:
You're missing the dependencies that a Tracker Query has here. The documentation says that it creates a space, but it doesn't because the dependencies are empty. You need to update the documentation and the dependencies as well because the dependencies are not correct. A `TrackerQuery` object needs 

1. a space, 
2. a work item type and 
3. a tracker.

See the `struct` definition:

```go
	// TrackerID is a foreign key for a tracker
	TrackerID uuid.UUID `gorm:"ForeignKey:TrackerID"`
	// SpaceID is a foreign key for a space
	SpaceID        uuid.UUID `gorm:"ForeignKey:SpaceID"`
	WorkItemTypeID uuid.UUID `gorm:"ForeignKey:WorkItemTypeID"`
```

Then you can use `NewTestFixture(t, DB, TrackerQuery(1))` and everything will be setup for you. Right now you need to manually specify `NewTestFixture(t, DB, Space(1), WorkItemType(1), Tracker(1), TrackerQuery(1))` which bothered me in another or this PR.